src/stores/useTimerStore.ts

```diff
@@ -0,0 +1,109 @@
+import { defineStore } from 'pinia';
+import { computed, reactive } from 'vue';
+
+type ConfigProps = {
+  pomodoro: number;
+  shortBreak: number;
+  longBreak: number;
+  longBreakInterval: number;
+};
+
+type TimerStoreProps = {
```

Comment:
```suggestion
type TimerStoreState= {
```

TimerStoreProps beschreibt den internen State, vllt wäre TimerStoreState treffender.